great_expectations/data_context/data_context/serializable_data_context.py

```diff
@@ -118,7 +118,7 @@ def _create(
         """
         Build a new great_expectations directory and DataContext object in the provided project_root_dir.
 
-        `create` will create a new "great_expectations" directory in the provided folder, provided one does not
+        `create` will create a new "gx" directory in the provided folder, provided one does not
```

Comment:
My editor complains when using f-strings in the docstring, so respecting that and just hard-coding it.